streaming/text/convert/README.md

```diff
@@ -2,16 +2,71 @@
-# For 'val' split, it will take anywhere from 10 sec to 1 min depending on your Internet bandwidth.
-# For 'train' split, it will take anywhere from 1-to-many hours depending on bandwidth, # CPUs, etc.
-# Once the script completes, you should see a dataset folder `./my-copy-c4/val` that is ~0.5GB and
-# `./my-copy-c4/train` this is ~800GB.
+## 1. [C4: Colossal, Cleaned, Common Crawl dataset](https://huggingface.co/datasets/c4)
 
-python c4.py --out_root ./my-copy-c4
-```
+**Instructions:**
+
+1. Run the [c4.py](c4.py) script as shown below with the minimalist arguments. The script downloads the raw format `train` and `val` split from HuggingFace hub and converts to StreamingDataset MDS format into their own split directory. For advanced users, you can look at the supported arguments for [c4.py](c4.py) and change according to your own needs.
+    <!--pytest.mark.skip-->
+    ```
+    python c4.py --out_root <local or remote directory path to save output MDS shard files>
+    ```
+
+## 2. [Wikipedia](https://huggingface.co/datasets/wikipedia)
+
+**Instructions:**
+
+1. Download English Wikipedia 2020-01-01 from [here](https://drive.google.com/drive/folders/1cywmDnAsrP5-2vsr8GDc6QUc7VWe-M3v).
+2. unzip the file `results_text.zip` as shown below.
+    <!--pytest.mark.skip-->
+    ```bash
+    unzip results_text.zip
+    ```
+
+    That will result in this directory structure:
+    <!--pytest.mark.skip-->
+    ```bash
+    ├── eval.txt
+    ├── part-00000-of-00500
+    ├── part-00001-of-00500
+    ├── part-00002-of-00500
+    ├── .....
+    ├── part-00498-of-00500
+    └── part-00499-of-00500
+    ```
+
+3. Run the [enwiki_text.py](enwiki_text.py) script as shown below with the minimalist arguments. The script converts the `train` and `val` dataset split into their own split directory. For advanced users, you can look at the supported arguments for [enwiki_text.py](enwiki_text.py) and change according to your own needs.
+    <!--pytest.mark.skip-->
+    ```
+    python enwiki_text.py --in_root <Above directory> --out_root <local or remote directory path to save output MDS shard files>
+    ```
+
+## 3. [Pile](https://pile.eleuther.ai/)
+
+**Instructions:**
+1. Download the Pile dataset from [here](https://the-eye.eu/public/AI/pile/).
+
+   That will result in this directory structure:
+    <!--pytest.mark.skip-->
+    ```bash
+    ├── SHA256SUMS.txt
+    ├── test.jsonl.zst
+    ├── train
+    │   ├── 00.jsonl.zst
+    │   ├── 01.jsonl.zst
+    │   ├── 02.jsonl.zst
+    │   ├── 03.jsonl.zst
+    │   ├── .....
+    │   ├── 28.jsonl.zst
+    │   └── 29.jsonl.zst
+    └── val.jsonl.zst
+    ```
+
+2. Run the [pile.py](pile.py) script as shown below with the minimalist arguments. The script converts the `train`, `test`, and `val` dataset split into their own split directory. For advanced users, you can look at the supported arguments for [pile.py](pile.py) and change according to your own needs.
```

Comment:
```suggestion
2. Run the [pile.py](pile.py) script with the minimalist arguments. The script converts the `train`, `test`, and `val` dataset splits into their own split directories. For more advanced use cases, yplease see the supported arguments for [pile.py](pile.py) and modify as necessary.
```

Reply:
Addressed as part of https://github.com/mosaicml/streaming/pull/183/commits/4d242aaf5f26f73278a555a3021da6e328d0a28b.